statements/uploaded_file.md

```diff
@@ -0,0 +1,35 @@
+# Uploaded a file
+
+## Description
+
+This event is emitted when a user has uploaded a file such as a pdf, doc, txt, ppt, xls, csv, etc.
+
+## Example
+
+```json
+{
+   "actor": {
+      "objectType": "Agent",
+      "account": {
+         "name": "john",
+         "homePage": "http://gaiax.org"
+      }
+   },
+   "verb": {
+      "id": "https://w3id.org/xapi/netc/verbs/uploaded"
+   },
+   "object": {
+      "objectType": "Activity",
+      "id": "http://gaiax.org/xapi/activities/ba297687-b1aa-4477-9efd-a782c8fdb90a",
+      "definition": {
+         "type": "http://activitystrea.ms/file",
+         "name": {
+            "en": "Test Course"
+         },
+         "extensions": {
+            "https://w3id.org/xapi/acrossx/extensions/type": "application/pdf"
+         }
+      }
+   }
+}
+```
```

Comment:
````suggestion
```
## Determining properties

| Property  | Value         |
|----------------|-----------------|
| verb.id | Must be `https://w3id.org/xapi/netc/verbs/uploaded` |
| object.definition.type | Must be `http://activitystrea.ms/file` |

## Rules

- `context.extensions.https://w3id.org/xapi/acrossx/extensions/type`: <INCLUDED> or <RECOMMENDED>, give information about the value(s).
````